buildSrc/src/main/kotlin/io/spine/internal/dependency/Grpc.kt

```diff
@@ -27,10 +27,10 @@
 package io.spine.internal.dependency
 
 // https://github.com/grpc/grpc-java
-@Suppress("unused")
+@Suppress("unused", "ConstPropertyName")
```

Comment:
Should we just turn off some checks for the `buildSrc` Kotlin sources? It seems they don't do much.